internal/juju/applications.go

```diff
@@ -1114,7 +1114,13 @@ func (c applicationsClient) ReadApplication(input *ReadApplicationInput) (*ReadA
 	usedResources := make(map[string]string)
 	for _, iResources := range resources {
 		for _, resource := range iResources.Resources {
-			usedResources[resource.Name] = strconv.Itoa(resource.Revision)
+			// The reason why we do it is to then use the state value for this resource
+			// instead of the revision number.
```

Comment:
```suggestion
			// For custom resources we special case the revision number so that subsequent runs of `terraform apply`
			// use the value in state rather than the revision number.
```

Reply:
Just some minor renaming, you can leave out stuff like "the reason we why do this" because its redundant.